spec/feature/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'user index' do
+  background do
+    @user = User.create(name: 'test', photo: 'test.jpg', bio: 'test', posts_counter: 0)
+
+    visit user_path
+  end
+
+  scenario 'I can see the username of all other users' do
```

Comment:
- [ ] some of your tests are failing, Kindly fix this
![image](https://user-images.githubusercontent.com/66097371/186248417-d9f9c2a9-fd27-4b1c-b773-96a1c34c5c14.png)